spec/ics-template.md

```diff
@@ -0,0 +1,53 @@
+---
+ics: (number)
+title: (short title)
+stage: (current process stage)
+category: (ics category)
+author: (primary & additional authors)
+created: (creation date)
+modified: (modification date)
+---
+
+## Synopsis
+
+(high-level description of and rationale for specification)
+
+## Specification
+
+(main part of standard document - not all subsections are required)
+
+### Motivation
```

Comment:
As I'm writing the cosmos signed message spec, I'm having trouble finding content that differs from `Synopsis` and `Specification.Motivation`.

Reply:
I think a more detailed justification could be useful sometimes - if not in that spec, just omit the motivation subsection.